app/controllers/brexit_landing_page_controller.rb

```diff
@@ -26,7 +29,8 @@ def show
 
   def taxon
     base_path = request.path
-    @taxon ||= Rails.cache.fetch("collections_content_items#{base_path}", expires_in: 10.minutes) do
+    content_item = ContentItem.find!(base_path)
+    @taxon ||= Rails.cache.fetch("collections_content_items#{base_path}", expires_in: content_item.content_item_data["cache_control"]["max_age"]) do
```

Comment:
I think this change is not needed, since we only want to set the Cache Control header to cache pages for the correct length of time as declared in the Content Store Cache-Control response header. We don't want to cache the responses from internal API calls differently.

Reply:
ok, I'll remove